distribution/zip/jballerina-tools/build.gradle

```diff
@@ -84,7 +84,6 @@ dependencies {
     dist 'com.fasterxml.jackson.core:jackson-core:2.8.6'
     dist 'com.fasterxml.jackson.core:jackson-annotations:2.8.0'
     dist 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.8'
-    dist 'com.google.api.grpc:proto-google-common-protos:1.17.0'
```

Comment:
I saw this dependency has been used in MySQL, Kafka, and oauth2. When we remove this here, it won't be available in the ballerina runtime. @niveathika, @ThisaruGuruge, and @ldclakmal, please note.

Reply:
I don't think this library is used in other modules. AFAIK this is only used in protobuf tool. will revert if its not

Reply:
This is not used in Kafka.

Reply:
This is not used in oauth2.

Reply:
This is not used in MySQL too